src/vivarium_cluster_tools/psimulate/cli.py

```diff
@@ -82,12 +82,29 @@ def psimulate():
     "configuration file.",
     callback=cli_tools.coerce_to_full_path,
 )
+# TODO: MIC-5234 - use this where other cli cluster options are being used
+@click.option(
+    "--logging-verbosity",
```

Comment:
There is no l flag for location and I didn't want to do something too similar to the normal pdb for the main process.

Reply:
What about `-s, --sim-verbosity` and then we could do `-vvv -sss`?

Reply:
Regardless, it's probably worth slighly tweaking the description of the normal verbosity `-vvv` to clarify that it's the verbosity of the runner and to use `sss` (or whatever you end up w/) for simulation-specific verbosity.

Reply:
I like sim verbosity. I think I would rather have a level as the input versus counting the s or sss like pdb that way it is very clear they are different.